examples/support/Cargo.toml

```diff
@@ -5,8 +5,8 @@ authors = ["Will Usher <willusher.life@gmail.com>"]
 
 [dependencies]
 image = "0.22.0"
-arcball = "1.0.0"
-cgmath = "0.17.0"
+arcball = {path = "../../../arcball" }
```

Comment:
I think the CI failures are from here where it's looking for arcball on a local path. I think we could increment it to use arcball 1.1.0, but are there some other changes you had locally that the rest of the PR changes relied on?

Reply:
Indeed, good catch. I did not remember that I have done this modification :) Sorry about that!